code/yarn.lock

```diff
@@ -6067,19 +6067,14 @@ __metadata:
   version: 0.0.0-use.local
   resolution: "@storybook/core-common@workspace:lib/core-common"
   dependencies:
-    "@babel/core": ^7.20.2
     "@storybook/node-logger": 7.0.0-beta.44
     "@storybook/types": 7.0.0-beta.44
-    "@types/babel__core": ^7.1.20
-    "@types/express": ^4.7.0
     "@types/mock-fs": ^4.13.1
     "@types/node": ^16.0.0
     "@types/picomatch": ^2.3.0
     "@types/pretty-hrtime": ^1.0.0
     chalk: ^4.1.0
-    esbuild: ^0.16.4
```

Comment:
`esbuild` is a peerDependency of `esbuild-register`

I recon the above is likely the reason some of those exists, though there's no doubt in my mind, some of them are likely unused.

Reply:
> with a simple search

This does not tell the whole story, and consider we do not have good tests in place to check for pnp/pnpm compatibility / package conformance.. removing dependencies is a risky business, and can cause old bugs to re-appear.

Thread carefully

Reply:
Ah, yes, thank you. I see now that `yarn` had a warning I overlooked:
```
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
```

Running that command showed:
```
➤ YN0000: p174e8 → @storybook/core-common@workspace:lib/core-common doesn't provide @types/esbuild to esbuild-register
➤ YN0000: p9d342 → @storybook/core-common@workspace:lib/core-common doesn't provide esbuild to esbuild-register
```

I added back `esbuild`. As far as I can see `@types/esbuild` does not exist, so that seems like a bug in `esbuild-register` specifying a package that doesn't exist. I'll just leave that as is since that's how it was before so should not cause any new issues.

I much prefer `pnpm` which would make this much easier to find :smile:

Reply:
> I much prefer pnpm which would make this much easier to find 😄

some day..